libandroid-navigation-ui/src/main/java/com/mapbox/services/android/navigation/ui/v5/instruction/InstructionView.java

```diff
@@ -342,6 +327,26 @@ public void setDistanceFormatter(DistanceFormatter distanceFormatter) {
     }
   }
 
+  /**
+   * Gets the sound button which is used for muting/unmuting, for uses such as adding listeners and
+   * hiding the widget.
```

Comment:
@Guardiola31337 I called them `widget`s because some of them include other little UI elements (SoundButton has the `soundChip` which briefly shows either `muted` or `unmuted` when you click the FAB) but I don't mind changing the naming to `button`